cetmix_tower_server/static/src/terminal/tower_terminal_action.scss

```
@@ -0,0 +1,154 @@
+        0 12px 30px rgba(15, 23, 42, 0.18);
+}
+
+.o_cx_tower_terminal_output {
+    margin: 0;
+    padding: 0;
+    border: 0;
+    background: transparent !important;
+    color: #f8fafc !important;
+    font: 500 0.92rem/1.55 "IBM Plex Mono", "Fira Code", monospace;
+    white-space: pre-wrap;
+    word-break: break-word;
```

Stylelint (17.7.0)

[error] Deprecated keyword "break-word" for property "word-break" (declaration-property-value-keyword-no-deprecated)

(declaration-property-value-keyword-no-deprecated)